package.json

```diff
@@ -474,8 +474,8 @@
         "git-url-parse": "^11.1.2",
         "moment": "^2.27.0",
         "simple-git": "^1.132.0",
-        "vscode-azureappservice": "^0.65.0",
-        "vscode-azureextensionui": "^0.35.1",
+        "vscode-azureappservice": "^0.73.0",
+        "vscode-azureextensionui": "^0.39.4",
```

Comment:
One way to reduce complexity is to not bump these packages so high. `createExperimentationService` was added in ui [0.36.1](https://github.com/microsoft/vscode-azuretools/wiki/vscode-azureextensionui-changelog#v0361---2020-10-22), which is all you really need.

You could also consider copying `createExperimentationService.ts` from the UI package to this repo so that we don't have to bump anything.

Reply:
That seems like an easy enough solution.  Would you be comfortable with shipping that if that were to be the only change?

Reply:
Doing a test pass would be the largest factor in terms of "would you be comfortable", but yes this would certainly help as well.